src/modules/auth/store/user/user.select.ts

```diff
@@ -9,7 +8,3 @@ export const selectIsLoggedIn = (state: AppState): boolean => !!state.user.user;
 export const selectHasAcceptedTosAt = (state: AppState): string | null => {
 	return state.user.user?.acceptedTosAt || null;
 };
-
-export const selectHasPermission = (state: AppState, permission: Permission): boolean => {
-	return !!state.user.user?.permissions.includes(permission);
-};
```

Comment:
Why remove this selector? You could still wrap it in a hook
```
export const selectHasAllPermissions = (state: AppState, permission: Permission): boolean => {
	return state.user.user && intersection(state.user.user.permissions.permissions, permissions).length === permissions.length;
};
```
```
export const useHasAllPermission = (...permissions: Permission[]): boolean => {
	return useSelector(selectHasAllPermissions);
};
```

Reply:
Since i created hooks specifically to get these permissions, we don't need the selector anymore. Better to just have 1 way to fetch permissions.
https://github.com/viaacode/hetarchief-client/pull/280/files#diff-71792b997b9f74a7d73fe70e8a0d8ce483adbf9a5bee1a09e900cfd2cbe910a0R11-R17